web/src/server_events_dispatch.js

```diff
@@ -470,6 +471,7 @@ export function dispatch_normal_event(event) {
                     settings_account.maybe_update_deactivate_account_button();
                     if (people.user_is_bot(event.person.user_id)) {
                         settings_users.update_bot_data(event.person.user_id);
+                        user_profile.update_bot_data(event.person.user_id);
```

Comment:
I'm a bit confused. Since the admin doesn't have the ability to update a user's avatar and email, why should we implement updates for these fields?
Are you suggesting that if a user updates their profile, the changes should also be reflected in the organization settings if the admin has the profile modal open simultaneously?
I attempted to implement this, but currently, any changes made by a user are only visible in the view profile tab and not in the manage user tab for the admin. 
Let me know if I've understood the it correctly, and I'll update the PR accordingly.

Reply:
Those fields can be updated in other ways outside the UI (LDAP sync, for example) -- and also a single user can have multiple browser tabs open. So it is most correct to have live-update work for all fields when the server tells the client about a new value.